spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -169,8 +211,9 @@ function onChanOpenAck(
   counterpartyChannelIdentifier: Identifier,
   counterpartyVersion: string) {
   // port has already been validated
-  // assert that counterparty selected version is "ics20-1"
-  abortTransactionUnless(counterpartyVersion === "ics20-1")
+  // assert that counterparty selected version is the same as our version
+  channel = provableStore.get(channelPath(portIdentifier, channelIdentifier))
+  abortTransactionUnless(counterpartyVersion === channel.version)
```

Comment:
@AdityaSripal I believe this approach does not work with middleware. For example, if I have fee middleware enabled with `ics20-2`, the `counterpartyVersion` will be the unwrapped string `ics20-2`, while the `channel.version` will be a json object which includes both the fee/transfer app versions.

Reply:
Discussed on slack, we can access the version via the `ics4Wrapper`